docs/src/rust-feature-support.md

```diff
@@ -200,3 +200,12 @@ related to [advanced features](#advanced-features).
 
 Please refer to [Intrinsics](rust-feature-support/intrinsics.md) for information
 on the current support in Kani for Rust compiler intrinsics.
+
+### Floating point operations
+
+Kani supports floating point numbers, but some kinds of operations on floats are "over-abstracted."
```

Comment:
I think we need to specify the exact set of operations. Otherwise, it may sound that none of the floating-point operations can be trusted, which is not the case. From looking at https://github.com/diffblue/cbmc/blob/16240afb9a855951533a57a0343b1737dd3f0168/src/ansi-c/library/math.c#L508, it seems only trigonometric functions and sqrt use abstractions (and consequently, any functions that call them).

Reply:
Hmm, I added a sentence saying this, but I tried to leave a tiny bit of wiggle room for there to be more operations. Without having deeply looked into it myself, I don't really have fully confidence we definitely know the exact set, and... well, it might change over time.